CMakeLists.txt

```diff
@@ -1,31 +1,25 @@
 #
-# Copyright (c) 2021-present, Trail of Bits, Inc.
+# Copyright (c) 2020-present, Trail of Bits, Inc.
 # All rights reserved.
 #
 # This source code is licensed in accordance with the terms specified in
 # the LICENSE file found in the root directory of this source tree.
 #
 
-# Fix behavior of CMAKE_CXX_STANDARD when targeting macOS.
-if (POLICY CMP0025)
-  cmake_policy(SET CMP0025 NEW)
-endif ()
-
-if (DEFINED VCPKG_ROOT)
-  message(WARNING "Building with VCPKG")
-  include(cmake/vcpkg_helper.cmake)
-else()
-  message(WARNING "Running the standard CMake build; if this is a mistake, reconfigure again and pass `-DVCPKG_ROOT=/path/to/vcpkg`")
-  set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
-endif()
+include("cmake/vcpkg_helper.cmake")
```

Comment:
What's the use of removing the option to build in a different way? I felt like the warning messages and README were good enough for notifying users that they are building one way or another and that vcpkg was the supported way.

Reply:
It means the need for additional logic, like fixing up z3 things. It means we can't make guarantees on the llvm version, etc. I would rather we require that we only support vcpkg-based builds.